interop/grpclb_fallback/client.go

```diff
@@ -1,4 +1,5 @@
 // +build !appengine
+// +build !darwin
```

Comment:
It seems this variable is ONLY defined for linux: https://github.com/golang/sys/search?p=1&q=TCP_USER_TIMEOUT&unscoped_q=TCP_USER_TIMEOUT

Maybe `build linux` instead?

Reply:
Done.